core/cas-server-core-web/src/main/java/org/apereo/cas/config/CasCoreWebConfiguration.java

```diff
@@ -30,6 +33,24 @@
     @Autowired
     private CasConfigurationProperties casProperties;
 
+    /**
+     * Get messages from one or more property files containing values
+     * that should be exposed to Thyme templates but don't need i18n.
+     * This bean is Autowired into {@link messageSource()}.
+     * @return PropertiesFactoryBean containing all common (non-i18n) messages
+     */
+    @Bean
+    public PropertiesFactoryBean casCommonMessages() {
```

Comment:
Is it not possible to simply not create this as a standalone bean and create it internally, injecting it into the reloadable bundle bean directly?